tests/unit-tests/src/VuFindTest/HttpServiceTest.php

```diff
@@ -474,7 +474,7 @@ public function testCurlAdapterDefaults()
         $adapter = $client->getAdapter();
         $this->assertInstanceOf('Laminas\Http\Client\Adapter\Curl', $adapter);
         $config = $adapter->getConfig();
-        $this->assertTrue(empty($config['curloptions']));
+        $this->assertFalse(isset($config['curloptions']));
```

Comment:
php-cs-fixer refactored this to `assertEmpty`, but that caused a warning since `$config['curloptions']` was unset. I revised the test to be more specific.